FlashPoint/src/controllers/victim_controller.py

```diff
@@ -59,8 +59,8 @@ def check_stop_lead(self, tile_model: TileModel) -> bool:
     def check_pickup(self, tile: TileModel) -> bool:
         game: GameStateModel = GameStateModel.instance()
 
-        if isinstance(self._current_player.carrying_hazmat, HazmatModel):
-            # Cant carry victim and hazmat at the same time
+        # Cannot carry a victim if already carrying a victim/hazmat
+        if isinstance(self._current_player.carrying_hazmat, HazmatModel) or isinstance(self._current_player.carrying_victim, VictimModel):
```

Comment:
Anything similar when picking up hazmats? If so we need to add the same thing to that portion of code

Reply:
The checks for picking up a hazmat check for both victims and hazmats.